.github/workflows/build_prs_jest_report.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: read
-      pull_requests: write
+      pull-requests: write
```

Comment:
Unrelated but just spotted this typo: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#example-setting-the-github_token-permissions-for-one-job-in-a-workflow